lib/main.ts

```diff
@@ -74,6 +86,9 @@ export class Transpiler {
     }
     var destinationRoot = destination || this.options.basePath || '';
     var program = ts.createProgram(fileNames, OPTIONS, host);
+    if (this.options.translateBuiltins) {
+      this.fc.setTypeChecker(program.getTypeChecker());
```

Comment:
Yeah, the transpiler steps are created before creating the program (they can be used multiple times over different programs), and I can only get the TypeChecker from the program, but not from the individual nodes we're visiting. The alternative would be passing down the TypeChecker with the program nodes, but that seems overkill.